package.json

```diff
@@ -17,11 +15,14 @@
     "list-stream": "~1.0.0",
     "minimist": "~1.1.0",
     "split2": "~0.2.1",
+    "strip-ansi": "^2.0.1",
```

Comment:
No `^` please, it needs to die in a fire. Also the changes introduced here are huge, perhaps something npm is doing now? very odd, but can you restore it to original and manually add in the `strip-ansi` line to make the impact less dramatic please?